lib/problem_check.rb

```diff
@@ -50,4 +52,19 @@ def self.call
   def call
     raise NotImplementedError
   end
+
+  private
+
+  def problem
+    [Problem.new(I18n.t(translation_key), priority: self.config.priority, identifier:)]
+  end
+
+  def no_problem
+    []
+  end
+
+  def translation_key
```

Comment:
Will probably also want `translation_args` here, or defined in another method, since e.g. `twitter_login_warning` needs a `base_path` probided

Reply:
Good catch!